frontend/model.py

```
@@ -0,0 +1,15 @@
+class User:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)